packages/lesswrong/server/utils/contributorsFieldHelper.ts

```diff
@@ -0,0 +1,74 @@
+import { forumSelect } from '@/lib/forumTypeUtils';
+import { getContributorsList, ContributorWithStats } from './contributorsUtil';
+import { loadByIds } from '@/lib/loaders';
+import { accessFilterMultiple } from '@/lib/utils/schemaUtils';
+import keyBy from 'lodash/keyBy';
+import orderBy from 'lodash/orderBy';
+import take from 'lodash/take';
+
+interface ContributorsFieldOptions {
+  collectionName: 'Tags' | 'MultiDocuments';
+  fieldName: string;
+}
```

Comment:
I'd type this as:
```
type ContributorsFieldOptions = {
  collectionName: 'Tags';
  fieldName: 'description';
} | {
  collectionName: 'MultiDocuments';
  fieldName: 'contents';
};
```

Since it's not like we don't know the legal/intended value of `fieldName` for each of those collections here.